pages/admin/admin-management.tsx

```diff
@@ -1,15 +1,140 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+// Eslint Disable Temporarily for Pagination since API hookup not complete.
 import { GetServerSideProps } from 'next'; // Get server side props
 import { getSession } from 'next-auth/client'; // Session management
-import { Text, GridItem } from '@chakra-ui/react'; // Chakra UI
 import Layout from '@components/internal/Layout'; // Layout component
 import { authorize } from '@tools/authorization'; // Page authorization
+import Table from '@components/internal/Table'; // Table component
 
+import {
+  Flex,
+  Text,
+  Box,
+  GridItem,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  IconButton,
+  Button,
+  Select,
+} from '@chakra-ui/react'; // Chakra UI
 // Internal home page
+import { AddIcon } from '@chakra-ui/icons';
+import Pagination from '@components/internal/Pagination'; // Pagination component
+
+function renderActionsMenu() {
+  return (
+    <Menu>
+      <MenuButton
+        as={IconButton}
+        aria-label="Options"
+        icon={<img src="/assets/three-dots.svg" />}
+        variant="outline"
+        border="none"
+      />
+      <MenuList>
+        <MenuItem>Edit User</MenuItem>
+        <MenuItem color="text.critical" textStyle="button-regular">
+          Delete User
+        </MenuItem>
+      </MenuList>
+    </Menu>
+  );
+}
+
+type RoleProps = {
+  readonly value: string;
+};
+
+function renderRoleMenu({ value }: RoleProps) {
```

Comment:
```suggestion
function renderRoleMenu({ value }: { value: string; }) {
```
And we can remove `RoleProps` above

Reply:
Also please add a comment to this function, for example:
```typescript
// Render the roles menu
function renderRoleMenu({ value }: { value: string; }) {
```

Reply:
bff1ca1f3ef4ba015e1df9e2edba2afb0946be92